plugin/oauth2/plugin.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
Codacy found an issue: [End of line character is invalid; expected "'\n'" but found "\n"](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5679956)